api-spec/protobuf/ark/v1/service.proto

```diff
@@ -222,9 +234,27 @@ message GetEventStreamResponse {
     TreeSignatureEvent tree_signature = 8;
     TreeNoncesEvent tree_nonces = 9;
     Heartbeat heartbeat = 10;
+    StreamStartedEvent stream_started = 11;
   }
 }
 
+// Adding and removing topics can both be supplied in the same request,
+// allowing for simultaneous changes.
+// overwrite_topics will take precedence, and if set, then the add/remove
+// topics are ignored. The stream_id is required.
+message UpdateStreamTopicsRequest {
+  string stream_id = 1;
+  repeated string add_topics = 2;
+  repeated string remove_topics = 3;
+  repeated string overwrite_topics = 4;
```

Comment:
Can we use oneOf here to make it clearer by definition?

Reply:
yes I like that, will add